tests/functionality/opflow/ipopt_pbpol.toml

```diff
@@ -272,3 +272,11 @@ is_powerimb_active =	true
 gen_bus_voltage_type = 'VARIABLE_WITHIN_BOUNDS'
 num_iters = 27
 obj_value = 740032.301353
+
+[[testcase]]
+network = 'datafiles/case9/case9_dcline.m'
+description = "Test for OPFLOW with DC line"
+is_loadloss_active =	false
+is_powerimb_active =	false
+num_iters = 18
+obj_value = 5312.16147
```

Comment:
This feels like a strict duplicate of the test you added in `tests/functionality/opflow/CMakeLists.txt`, but I realize we still have a few functionality tests duplicated in this format, so probably not the biggest deal.

Reply:
Yes, you are right. This is a duplicate.